x-pack/plugins/log_explorer/public/customizations/log_explorer_profile.tsx

```diff
@@ -123,6 +123,7 @@ export const createLogExplorerProfileCustomizations =
      */
     customizations.set({
       id: 'flyout',
+      size: '60%',
```

Comment:
what about reset ? Will it not impact the Flyout on the Discover page also if we only set it for Logs Explorer ?

Reply:
A reset is not required here since this is a customization property of the flyout extension point for the `observability-log-explorer` profile.

When the user navigates to Discover, the customization framework will see that page as the `default` profile and apply the related values, so it's not necessary to reset the size, you can think of it as the `title` or `Content` property work.